webpack.config.js

```diff
@@ -174,8 +174,8 @@ module.exports = async (env) => [
               or: [
                 (input) => posixPath(input).includes("node_modules/@patternfly/react-core/dist/styles/assets/fonts"),
                 (input) => posixPath(input).includes("node_modules/@patternfly/react-core/dist/styles/assets/pficon"),
-                (input) => posixPath(input).includes("node_modules/@patternfly/patternfly/assets/fonts"),
-                (input) => posixPath(input).includes("node_modules/@patternfly/patternfly/assets/pficon"),
```

Comment:
reported https://github.com/KaotoIO/vscode-kaoto/issues/69 to investigate if we can get rid of completely of these lines which are having a very coupling with transitive dependencies